app/controllers/chargeback_controller.rb

```diff
@@ -305,11 +305,16 @@ def cb_rates_delete
         end
       end
       process_cb_rates(rates, "destroy")  unless rates.empty?
-      add_flash(_("The selected %{records} were deleted") %
-        {:records => ui_lookup(:models => "ChargebackRate")}, :info, true) unless flash_errors?
-      cb_rates_list
-      @right_cell_text = _("%{typ} %{model}") % {:typ => x_node.split('-').last, :model => ui_lookup(:models => "ChargebackRate")}
-      replace_right_cell([:cb_rates])
+      if flash_errors? && @flash_array.count == 1
```

Comment:
@h-kataria @dclarizio if you are clear on the need for this check, I am :+1: on the rest of this PR